crates/core/tedge_mapper/src/main.rs

```diff
@@ -111,11 +124,88 @@ async fn main() -> anyhow::Result<()> {
         )?);
     }
 
+    let mut mqtt_actor = get_mqtt_actor(component.session_name(), &config).await?;
+    let mut runtime = Runtime::try_new(runtime_events_logger).await?;
+    let mut signal_actor = SignalActor::builder();
```

Comment:
These initialisations are needed only inside the `else` block where the mappers are really run. Else they'll be initialized unnecessarily for `init` and `clear` as well.